manic/repository_git.py

```diff
@@ -687,37 +712,91 @@ def _git_remote_verbose():
         git_output = execute_subprocess(cmd, output_to_caller=True)
         return git_output
 
+    @staticmethod
+    def _git_version():
+        """Run the git --version command to obtain the git version.
```

Comment:
Can you add to this documentation an example illustrating the format of the returned value?

Reply:
I think I can get rid of this.